doc/changelog.md

```diff
@@ -3,6 +3,52 @@
 <h3>New features</h3>
 
 <h3>Improvements</h3>
+* Added the PSWAP and ISWAP gates to refer to the Amazon Braket pswap and iswap gates in the backend. Specifically, a circuit like
+ 
+  ```py
+  import pennylane as qml
+  from catalyst import *
+
+  dev = qml.device("braket.local.qubit", wires=2, shots=100)
+
+  @qjit
+  @qml.qnode(dev)
+  def f(x: float):
+      qml.Hadamard(0)
+      qml.PSWAP(x, wires=[0, 1])
+      qml.ISWAP(wires=[1, 0])
+      return qml.probs()
+  ```
+
+would no longer decompose the PSWAP and ISWAP gates into braket backend's SWAPs and CNOTs and Hadamards. Instead it would just call braket backend's PSWAP and ISWAP:
```

Comment:
```suggestion
would no longer decompose the `PSWAP` and `ISWAP` gates to `SWAP`s, `CNOT`s and `Hadamard`s. Instead it would just call Braket's native `PSWAP` and `ISWAP` gates at runtime.
```